python/src/RandomWalkMetropolisHastings_doc.i.in

```diff
@@ -164,22 +164,32 @@ expansionFactor : float, :math:`e > 1`
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::RandomWalkMetropolisHastings::getAdaptationRange
-"Get the range.
+"Get the expected range for the acceptance rate.
+During burn-in, at the end of every adaptation period,
+if the acceptance rate does not belong to this range,
+the adaptation factor is multiplied
+either by the expansion factor (if the acceptance rate is too high)
```

Comment:
What exactly means "too high". Is there a threshold? What is the parameter that we can use to set that threshold?

Reply:
Yes, the thresholds are set in the ResourceMap. This is now explicitely stated.